### DISCUSSION TITLE
New class for IT/computer/network related metering

### DISCUSSION THREAD

**Discussion by grodansparadis** [grodansparadis/vscp#267] [Ideas]
I plan to add a new class for IT/computer/network related metering. Typically "disk space", "memory", "load", "bandwidth" and such types. 

Please if you have suggestions for types  add them here.  Also any comments are welcome of course.

**Comment by BlueAndi** [comment]
latency

**Comment by BlueAndi** [comment]
system informations, like os, version, ip, mac, etc. too?
